index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="sideNav">
+        <h1>JS documentation</h1>
+        <ul>
+          <li><a href="#intro">Introduction</a></li>
+          <li><a href="#"></a></li>
+          <li><a href="#what">What you should already know</a></li>
+          <li><a href="#js">Javascript or Java</a></li>
+          <li><a href="#hello">Hello world</a></li>
+          <li><a href="#variable">Variable</a></li>
+          <li><a href="#declare">Declaring variable</a></li>
+          <li><a href="#scope">Variable scope</a></li>
+          <li><a href="#global">Global variables</a></li>
+          <li><a href="#constant">Constants</a></li>
+          <li><a href="#data">Data types</a></li>
+          <li><a href="#condition">if...else statement</a></li>
+          <li><a href="#function">Function declarations</a></li>
+          <li><a href="#refrence">Refrence</a></li>
+        </ul>
+      </div>
```

Comment:
You didn't link the CSS file. Do it right now!!!!! 😠

Reply:
I think it is better to convey your message in a polite and inviting manner. There are many ways to rephrase this feedback to make it more polite, my suggestion is as follows: Excellent work on working with this project. As you have noticed, you have missed to link your CSS file in HTML . Kindly fix this.